docs/book/generator/examples.md

```diff
@@ -370,3 +370,46 @@ $generator->addMethod(
 );
 $code = $generator->generate();
 ```
+
+
+## Generating Trait 
+
+`Laminas\Code\Generator\FileGenerator` can be used to generate the contents of a *PHP* file. You can
+include classes as well as arbitrary content body.
```

Comment:
These two sentences are copied from the [section "Generating PHP files"](https://docs.laminas.dev/laminas-code/generator/examples/#generating-php-files) and it does not the describe the generating of a trait.